src/controller/cards_controller.js

```diff
@@ -1,4 +1,9 @@
+const config = require('config');
+
 const cardsController = (Card) => {
+    const host = config.get('server.host');
```

Comment:
is it posible to use the deconstruct here? `const { host, port } = config.get('server');`

Reply:
yes it is, I'll make the changes